org.eclipse.xtext.web.example.statemachine/src-gen/org/eclipse/xtext/web/example/statemachine/services/StatemachineGrammarAccess.java

```diff
@@ -43,7 +43,8 @@
 		//	states+=State*;
 		@Override public ParserRule getRule() { return rule; }
 		
-		//{Statemachine} signals+=Signal* states+=State*
+		//{Statemachine} signals+=Signal*
```

Comment:
Unrelated: I wonder if we should use the verbatim rule body instead.
Why: The statemachine example grammar looks like being carefully formatted by hand. I suppose this is true for most languages - the grammar is usually written such that the developer perceives it as well formatted. Why do we bother using the formatter here?